README.md

```diff
@@ -13,6 +13,8 @@ Currently, we have pre-built binaries for macOS (both Intel and Apple Silicon),
 
 `pip install viam-sdk`
 
+If you are using conda as a virtual environment & MacOS version 11 or later, you will need to run `export SYSTEM_VERSION_COMPAT=0` before installing `pip install viam-sdk.` This is because of a bug in how conda tags MacOS 11. [See more](https://github.com/conda-forge/python-feedstock/issues/445)
```

Comment:
Some nits:
* `macOS` with a lowercase `m`
* "you MAY need" instead of 'you WILL need"
* You can run it all in one line `SYSTEM_VERSION_COMPAT=0 pip install viam-sdk`